api/src/main/java/org/openmrs/api/handler/PatientDataVoidHandler.java

```diff
@@ -60,12 +60,11 @@ public void handle(Patient patient, User voidingUser, Date voidedDate, String vo
 			}
 		}
 
-		Context.addProxyPrivilege(PrivilegeConstants.EDIT_COHORTS);
+		
 		try {
 			Context.getCohortService().notifyPatientVoided(patient);
 		}
 		finally {
-			Context.removeProxyPrivilege(PrivilegeConstants.EDIT_COHORTS);
```

Comment:
Another one

Reply:
Still working on the travis failure and then clean up the code.